app/services/dataset_file_schema_service.rb

```diff
@@ -1,19 +1,20 @@
 class DatasetFileSchemaService
 
-  def initialize(schema_name, description, url_in_s3, user, owner_username = user.name)
+  def initialize(schema_name, description, url_in_s3, user, owner_username = user.name, restricted = true)
     @schema_name = schema_name
     @description = description
     @url_in_s3 = url_in_s3
     @user = user
     @owner_username = owner_username
+    @restricted = restricted
```

Comment:
does this mean schemas already in the database will have the restricted flag disabled by default?